pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -108,6 +109,10 @@ func (admitter *VirtualMachineCloneAdmitter) Admit(ar *admissionv1.AdmissionRevi
 		causes = append(causes, newCauses...)
 	}
 
+	if newCauses := validateNewMacAddresses(vmClone); newCauses != nil {
+		causes = append(causes, newCauses...)
```

Comment:
You can simply write this:
```suggestion
		causes = append(causes, validateNewMacAddresses(vmClone)...)
```

Reply:
Shouldn't we be consistent with the above?

Reply:
done